queue_services/entity-filer/src/entity_filer/filing_processors/continuation_out.py

```diff
@@ -0,0 +1,58 @@
+# Copyright © 2023 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""File processing rules and actions for the consent continuation out filing."""
+from contextlib import suppress
+from datetime import datetime
+from typing import Dict
+
+import dpath
+from legal_api.models import Business, Filing
+
+from entity_filer.filing_meta import FilingMeta
+from entity_filer.filing_processors.filing_components import filings
+
+
+def process(business: Business, continuation_out_filing: Filing, filing: Dict, filing_meta: FilingMeta):
```

Comment:
some other things you will need to address in this function:
* the `business.state_filing_id` to be the filing id of the continuation out filing.
* is the detail comment added as a filing comment?  don't think this is addressed yet

Reply:
Hi @argush3  @vysakh-menon-aot 
But, I cannot see the comments property in ContinuationOut Schema. - https://github.com/bcgov/business-schemas/blob/main/src/registry_schemas/schemas/continuation_out.json

Reply:
`continuation_out_json.get('details')` should be saved in `Comment` table (refer correction code mentioned above) instead of `continuation_out_filing.order_details = details` in line no 41